### ISSUE TITLE
Flaky test at creating course - Gather debug info

### ISSUE THREAD

**Issue by javiertuya** [giis-uniovi/retorch-st-fullteaching#49]
Past week a dependabot update failed many TJobs. This was during the host daily live backup, that can justify the failure because of the very high disk activity during the backup window.

But today, when running the combined update  at PR #48, it failed again. Now, only a TJob with message: `Failed to create course:: class com.fullteaching.e2e.no_elastest.common.exception.ElementNotFoundException: newCourse - Course hasn't been created`. Although this was running in parallel with other jobs, it seems that it does not justify the flakiness.

Before I rerun and merge the update, please, gather all needed data, including videos for further debug, and tell me when it is done

**Comment by javiertuya** [comment]
@augustocristian Is this already done?

**Comment by javiertuya** [comment]
waiting to updates recreation to close
